cypress/integration/collections_spec.js

```diff
@@ -0,0 +1,83 @@
+describe('Dashboard Collections Page', () => {
+  const host = process.env.DASHBOARD_HOST || 'http://localhost:3000/';
+
+  describe('When not logged in', () => {
+    it('should redirect to login page', () => {
+      cy.visit(`${host}#/collections`);
+      cy.url().should('include', '/#/auth');
+      cy.get('div[class=modal__internal]').within(() => {
+        cy.get('a').should('have.attr', 'href').and('include', 'token?');
+        cy.get('a').should('have.text', 'Login with Earthdata Login');
+      });
+    });
+  });
+
+  describe('When logged in', () => {
+    beforeEach(() => {
+      cy.login(host);
+    });
+
+    it('displays a link to view collections', () => {
+      cy.visit(host);
+
+      cy.get('nav li a').contains('Collections').should('exist').as('collections');
+      cy.get('@collections').should('have.attr', 'href', '#/collections');
+      cy.get('@collections').click();
+
+      cy.url().should('include', 'collections');
+      cy.get('.heading--xlarge').should('have.text', 'Collections');
+
+      cy.get('table tbody tr').its('length').should('be.gt', 1);
+    });
+
+    it('collections page displays a button to add a new collection', () => {
+      cy.visit(`${host}/#collections`);
+
+      cy.get('.heading--large').should('have.text', 'Collection Overview');
+      cy.get('a').contains('Add a Collection').should('exist').as('addCollection');
+      cy.get('@addCollection').should('have.attr', 'href', '#/collections/add');
+      cy.get('@addCollection').click();
+
+      // fill the form and submit
+      // we need to use fake collection under test/fake-api-fixtures/collections, so the
+      // verification would work
+      const collection = '{{}"name":"MOD09GQ","version":"006","dataType":"MOD09GQ"}';
+      cy.get('textarea').type(collection, {force: true});
+      cy.get('form').get('input').contains('Submit').click();
+
+      // displays the new collection
+      cy.get('.heading--xlarge').should('have.text', 'Collections');
+      cy.get('.heading--large').should('have.text', 'MOD09GQ / 006');
+      cy.url().should('include', '#/collections/collection/MOD09GQ/006');
```

Comment:
Does this not work as `cy.url().should('equal', '#/collections/collection/MOD09GQ/006')`?

Reply:
the `url()` starts with host

Reply:
OK. I think that will change once #497 is merged